### ISSUE TITLE
Modify wall, roof, and floor conductance by HDD and climate zone.

### ISSUE THREAD

**Issue by MarianaNRC** [canmet-energy/btap_tasks#130]
Adjust the wall, roof, and floor conductance by the heating degree days.

**Comment by IainMacdonald** [comment]
This is a measure that will set the R-value of the envelope according to the HDD. Will be used to test options for TG-BE.

**Comment by IainMacdonald** [comment]
Which all should be used to determine the HDD? My understanding is that its not necessarily the HDD in the climate file but rather the HDD in table C1 of the NBC that should be used. Can someone confirm?

**Comment by IainMacdonald** [comment]
Created new branch nrc_130 for this activity.

**Comment by phylroy** [comment]
@IainMacdonald if you want an answer from somebody on this task.. you should ask a person.. Nobody will look at this unless they need to dig into it.

**Comment by phylroy** [comment]
As for the HDD days.. You should follow the NECB values. We already do this in the construction methods.

**Comment by IainMacdonald** [comment]
Is there a specific way to do that (ask a person)? I was also marking this down as an issue that I though could be discussed at the stand up as often I’m not sure who to specifically ask (should maybe just throw it out on skack?).

**Comment by IainMacdonald** [comment]
Windows added to measure

**Comment by IainMacdonald** [comment]
TO DO.
Mariana to upload measure onto branch nrc_130 in her fork.
Code walk through with NRCan
Adam to follow NRCan process for merging into canmet repository

**Comment by IainMacdonald** [comment]
Measure works and functionality has been included in BTAP_envelope_construction_measure. Next steps:
1) Confirm that BTAP_envelope_construction_measure works for NRC's needs
2) Delete our measure

**Comment by IainMacdonald** [comment]
@MarianaNRC can you check if the BTAP measure provides the same functionality

**Comment by MarianaNRC** [comment]
@IainMacdonald Yes ,  BTAP_envelope_construction_measure works. I've tested it for all 16 archetypes and 6 locations in OS server 2.6. I'll re-test for 2.8.1

**Comment by IainMacdonald** [comment]
@MarianaNRC @edwardv720 
So we can delete measures: btap_set_*_conductance_by_NECB_climate_zone (where * is fenestration, wall or roof)? i.e. all three measures.

**Comment by MarianaNRC** [comment]
@IainMacdonald , yes for me as everything is included in the construction measure.

**Comment by edwardv720** [comment]
@IainMacdonald  I'm ok with deleting the measure as well.

**Comment by IainMacdonald** [comment]
Pull request made to Phylroy. Updated code on banch nrc_130
